client/src/partials/subsidies/subsidies.html

```diff
@@ -118,40 +118,41 @@
               </bh-account-select>
 
               <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.description.$invalid }">
-                <label class="control-label" for="description">{{ 'FORM.LABELS.DESCRIPTION' | translate }}</label>
+                <label class="control-label" for="description" translate> FORM.LABELS.DESCRIPTION </label>
                 <textarea class="form-control" id="description" name="description" ng-maxlength="SubsidyCtrl.maxLength" ng-model="SubsidyCtrl.subsidy.description" rows="3"></textarea>
                 <div class="help-block" ng-messages="CreateForm.description.$error" ng-show="CreateForm.$submitted">
                   <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
                 </div>
               </div>
 
               <div class="form-group">
-                <button class="btn btn-default" id="submit-subsidy" type="submit" data-method="submit">
-                  {{ "FORM.BUTTONS.SUBMIT" | translate }}
-                </button>
-                <button class="btn btn-default" type="button" ng-click="SubsidyCtrl.cancel()" data-method="cancel">
-                  {{ "FORM.BUTTONS.CANCEL" | translate }}
+                <bh-loading-button loading-state="CreateForm.$loading">
+                  <span translate>FORM.BUTTONS.SUBMIT</span>
+                </bh-loading-button>
+
+                <button class="btn btn-default" type="button" ng-click="SubsidyCtrl.cancel()" data-method="cancel" translate>
+                  FORM.BUTTONS.CANCEL
                 </button>
               </div>
             </form>
           </div>
         </div>
 
         <div class="panel panel-primary" ng-switch-when="update">
-          <div class="panel-heading">{{ 'SUBSIDY.UPDATING_SUBSIDY' | translate }}</div>
+          <div class="panel-heading" translate> SUBSIDY.UPDATING_SUBSIDY </div>
           <div class="panel-body">
             <form name="UpdateForm" ng-submit="SubsidyCtrl.submit(UpdateForm.$invalid)" bh-form-defaults novalidate>
```

Comment:
For the `$loading` state to work, this must use `bh-submit` instead of `ng-submit`.